### DISCUSSION TITLE
Syntax completion in R markdown file .rmd?

### DISCUSSION THREAD

**Discussion by IsoRatio** [posit-dev/positron#3872] [Q&A]
Hi all,
might be a stupid question but I found no solutions: I want to have the syntax auto-complete function in Positron as RStudio. The wiki says the REditorSupport has been fully realized in Positron but I observe only auto-completion in console but not in .rmd file. Every time I tab, it says "no suggestions". Any hints for this?

**Comment by juliasilge** [comment]
Can you share more about what version of Positron you are using? The code completions in an `.Rmd` file are generated via Quarto (with some caveats, as outlined in #1572); can you double check that the Quarto extension is correctly installed/bundled? You should see this when you search for Quarto under Extensions:

![Screenshot 2024-07-05 at 1 09 01 PM](https://github.com/posit-dev/positron/assets/12505835/4b2877b2-c2ee-4fb1-a822-2784d2627b93)
